modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributor.java

```diff
@@ -236,6 +238,11 @@ protected String getItemSelectorURL(
 			return null;
 		}
 
+		long groupId = GetterUtil.getLong(
+			ddmFormFieldRenderingContext.getProperty("groupId"));
+
+		Group group = _groupLocalService.fetchGroup(groupId);
```

Comment:
@carolmariaabb, is there any chance of this group be `null`? If so, let's handle this scenario.